src/commands/appSettings/connectionSettings/eventHubs/IEventHubsConnectionWizardContext.ts

```diff
@@ -3,23 +3,17 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import type { AuthorizationRule, EHNamespace } from "@azure/arm-eventhub";
-import type { ResourceGroup } from "@azure/arm-resources";
+import type { ResourceManagementModels } from "@azure/arm-resources";
 import { ISubscriptionContext } from "@microsoft/vscode-azext-utils";
 import { EventHubsConnectionTypeValues, StorageConnectionTypeValues } from "../../../../constants";
 import { ISetConnectionSettingContext } from "../ISetConnectionSettingContext";
 
 export interface IEventHubsConnectionWizardContext extends ISetConnectionSettingContext, Partial<ISubscriptionContext> {
-    resourceGroup?: ResourceGroup;
+    resourceGroup?: ResourceManagementModels.ResourceGroup;
```

Comment:
Should the types be more like this?

![image](https://github.com/microsoft/vscode-azurefunctions/assets/40250218/9c7af20b-be11-49bb-85f5-ff9a493166cd)

Reply:
Do you mean the spaces? I'm not sure what you're trying to highlight here.
Or the fact that it has `resourceGroup` as a property?